doc/sphinx/source/recipes/recipe_seaice_extents_sh.rst

```diff
@@ -0,0 +1,72 @@
+.. _recipes_seaice_extents_sh:
+
+Sea Ice area and extents
+========================
+
+Overview
+--------
+
+This recipe plots sea ice concentration from CICE (sea ice model) output 
+around the southern polar region and compares it to the NSIDC CDR 
+(National Snow and Ice Data Centre, Climate Data Record) dataset.
+
+
+Available recipes and diagnostics
+---------------------------------
+
+Recipes are stored in esmvaltool/recipes/
+
+    * recipe_seaice_extents_sh.yml
+
+Diagnostics are stored in esmvaltool/diag_scripts/seaice_area_extents/
+
+    * seaicearea_trends.py: plot minima and maxima sea ice area trends
+    * seaice_mapextents.py: plot sea ice extent and differences to Observations
+
+
+User settings in recipe
+-----------------------
+
+#. Script seaice_mapextents.py
+
+   *Required settings for script*
+
+   * `months`: months by month number which the mean are to be plotted
+
```

Comment:
Typically we would specify the same time period for the model and the observation output when making the difference plots, which could be a setting.

1979 to 2008 could be a could choice. Lots of work aligns it to decades though, e.g. 1981 to 2010

Reply:
time periods can be edited in the yaml recipe file